daemon/containerd/image.go

```diff
@@ -68,24 +68,6 @@ func (i *ImageService) GetImage(ctx context.Context, refOrID string, options ima
 		exposedPorts[nat.Port(k)] = v
 	}
 
-	derefTimeSafely := func(t *time.Time) time.Time {
```

Comment:
LOL, didn't realise this would no longer be needed with your PR. Guess it's still fine as the other PR was very focussed on just this issue, and "got us there" to fix that case.

Reply:
Ohh, I somehow completely missed this PR and the fact that it would fix this 😄

Reply:
@vvoland no harm done! On the contrary: your fix was good, because it was small enough to backport, whereas this PR touches a lot more code, and unlikely would have qualified for inclusion in a patch release.